tests/api.test.js

```diff
@@ -42,6 +42,7 @@ describe('api', () => {
         expect(createCallResponse.result.applicationId).toEqual(applicationId);
         expect(createCallResponse.result.to).toEqual(to);
         expect(createCallResponse.result.from).toEqual(from);
+        expect(createCallResponse.result.enqueuedTime).toBeTruthy();
```

Comment:
What is `toBeTruthy()` validating?

Reply:
This could probably be toBeInstanceOf(Date)

Reply:
Since this is a breaking change anyways @dyeam-bw should we change all instances of timestamps to be datetime instead of strings for this release?

Reply:
`toBeTruthy()` validates against the stuff Brian said, and `toBeInstanceOf(String)` fails because the `string` is not an instance of the `String` object. I'd be fine with changing this to `Date`, but we definitely need to see what @dyeam-bw says.

Reply:
the test now tests `Date.parse(enqueuedTime)` `toBeTruthy()`. `Date.parse()` returns `NaN` if the string is empty or cannot be parsed, so this should test against both the string being null/empty and the string being formatted incorrectly.